src/DocumentWatcher.ts

```diff
@@ -172,6 +173,22 @@ function applyEditorConfigToTextEditor(
 	/* tslint:disable:no-any */
 	textEditor.options = <any> newOptions;
 	/* tslint:enable */
+
+	applyEndOfLineSetting();
+
+	function applyEndOfLineSetting() {
+		for (const [regex, eol] of new Map([
+			[/^lf$/i, EndOfLine.LF],
+			[/^crlf$/i, EndOfLine.CRLF]
+		])) {
+			if (regex.test(editorconfig.end_of_line)) {
+				textEditor.edit(edit => {
+					edit.setEndOfLine(eol);
+				});
+				return;
+			}
+		}
```

Comment:
Yeah. I can `toLowerCase()`. I believe it transforms all the line endings too. This isn't going to work like it should. That's why this is a WIP!